src/SnippetRendererBundle/objects/TPkgSnippetRenderer.class.php

```diff
@@ -146,9 +153,18 @@ public function render()
             $this->setFilename($this->getSourceModule()->viewTemplate);
         }
 
+        // TODO why didn't the old handling swallow up any "missing template" error?
```

Comment:
This is probably a valid question: Because now there will be a full exception on page and also everything afterwards is not rendered.

For example a "ArticleListLegacyMapper Unexpected error: Template "/common/teaser/standard-shopArticleExtendedWithQuickshopping.html.twig" is not defined"